geode-core/src/test/java/org/apache/geode/management/bean/stats/AsyncEventQueueStatsJUnitTest.java

```diff
@@ -23,8 +23,9 @@
 import org.apache.geode.cache.asyncqueue.internal.AsyncEventQueueStats;
 import org.apache.geode.management.internal.beans.AsyncEventQueueMBeanBridge;
 import org.apache.geode.test.junit.categories.IntegrationTest;
+import org.apache.geode.test.junit.categories.StatisticsTest;
 
-@Category(IntegrationTest.class)
+@Category({IntegrationTest.class, StatisticsTest.class})
```

Comment:
The statistics related tests under management should probably have category JMXTest with or without StatisticsTest